docs/tutorial/add_an_http_request.rst

```diff
@@ -1,4 +1,10 @@
 Add an HTTP request
 ===================
 
-This tutorial is under constructions.
+In this section we will show you how to add an http request to your sandbox application.
+
+As an example, we are going to preform a GET request to `CTA train tracker <http://lapi.transitchicago.com/api/1.0/ttpositions.aspx?key=33305d8dcece4aa58c651c740f88d1e2&rt=red&outputType=JSON>`_. which returns a total number of in-service trains for Red line route.
+
+.. figure::  images/http_request.png
```

Comment:
Can you explain a bit more on what's happening?
i.e The HTTP request is made to that URL, with the parameters `key`, `rt`  etc.. and the output type as Json.
Then also explain what you do with the result i.e what happens in the `.then()` function